src/components/Dropdown/Dropdown.tsx

```diff
@@ -155,17 +159,19 @@ export default class Dropdown extends AutoControlledComponent<
     fluid: PropTypes.bool,
     getA11ySelectionMessage: PropTypes.object,
     getA11yStatusMessage: PropTypes.func,
+    indicator: PropTypes.bool,
```

Comment:
maybe personal thing, but for me this name makes it being much less intuitive what it is responsible for